src/main/java/core/basesyntax/dao/jdbc/DaoJdbcCarImpl.java

```diff
@@ -0,0 +1,169 @@
+package core.basesyntax.dao.jdbc;
+
+import core.basesyntax.dao.DaoCar;
+import core.basesyntax.exceptions.DataProcessingException;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Car;
+import core.basesyntax.model.Driver;
+import core.basesyntax.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@SuppressWarnings("unused")
+@Dao
+public class DaoJdbcCarImpl implements DaoCar {
+    @Override
+    public Car create(Car item) {
+        String query = "INSERT INTO cars (model, manufacturer_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, item.getModel());
+            statement.setLong(2, item.getManufacturer().getId());
+            statement.execute();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                item.setId(resultSet.getObject("id", Long.class));
+            }
+            addDriversIntoCar(item.getId(), item.getDrivers(), connection);
+            return item;
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("ERROR: can't add car " + item, throwables);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT * FROM cars JOIN manufacturers m on cars.manufacturer_id = m.id "
+                + "WHERE m.deleted = false AND cars.deleted = false AND cars.id = ?";
```

Comment:
try to avoid `*` in SELECT statement, because it's the bad practice
```suggestion
        String query = "SELECT * FROM cars JOIN manufacturers m on cars.manufacturer_id = m.id "
                + "WHERE cars.deleted = false AND cars.id = ?";
```
also car can exist if we delete manufacturer